src/database/models/comment.js

```diff
@@ -0,0 +1,49 @@
+const comments = (sequelize, DataTypes) => {
+  const Comments = sequelize.define(
+    'Comments',
```

Comment:
Model should have singular name to maintain consistency within the project, please check here https://github.com/andela/ah-kifaru-backend/blob/develop/src/database/models/follower.js

Hence, when creating a comment table use: =>`cmd to create table` comment<=

Reply:
This has been solved on the comment branch